target/linux/lantiq/Makefile

```diff
@@ -9,7 +9,7 @@ BOARDNAME:=Lantiq
 FEATURES:=squashfs
 SUBTARGETS:=xrx200 xway xway_legacy falcon ase
 
-KERNEL_PATCHVER:=5.4
+KERNEL_PATCHVER:=5.10
 KERNEL_TESTING_PATCHVER:=5.10
```

Comment:
@adschm The disadvantage of keep this is that the target has the tag `testing kernel` while in fact stable and testing kernel are the same. If I wanted to setup a CI that continuously builds testing Kernels it would wrongly detect *lantiq* as a device with a testing kernel. In my opinion this variable should be removed until Kernel 5.15 is added (as a testing Kernel).

Reply:
Correction, the tag is only added if default != testing Kernel so @adschm approach is fine. For me it looks a bit odd to have both lines but I have no strong opinion since the tag is no longer an argument.

Reply:
Fine with me, I just remember a general mood against *useless* variables. Anyway let's stick with your approach for now.

Reply:
Note that I've merged "your" 5.10 patches with removal of the variable. I don't really think this is worth a discussion...